common/ledger/blkstorage/blockstore.go

```diff
@@ -93,7 +93,7 @@ func (store *BlockStore) RetrieveBlockByTxID(txID string) (*common.Block, error)
 }
 
 // RetrieveTxValidationCodeByTxID returns the validation code for the specified txID
-func (store *BlockStore) RetrieveTxValidationCodeByTxID(txID string) (peer.TxValidationCode, error) {
+func (store *BlockStore) RetrieveTxValidationCodeByTxID(txID string) (peer.TxValidationCode, uint64, error) {
```

Comment:
This is a change to a public method, but given Fabric is not designed to be a library, I don't think we need to document the API change.

Reply:
Good Catch! I'll fix this in a separate PR to avoid delay merging in this.